src/RokuDeploy.ts

```diff
@@ -919,7 +919,11 @@ export const DefaultFiles = [
     'source/**/*.*',
     'components/**/*.*',
     'images/**/*.*',
-    'manifest'
+    'manifest',
+    'locale/**/*',
+    'fonts/**/*',
+    '!node_modules',
+    '!**/*.{md,DS_Store,db}'
```

Comment:
Let's reorder these a bit to improve readability. 

export const DefaultFiles = [
    'source/**/*.*',
    'components/**/*.*',
    'images/**/*.*',
    'locale/**/*',
    'fonts/**/*',
    'manifest',
    '!node_modules',
    '!**/*.{md,DS_Store,db}'
]

Reply:
done